slices/openjdk-17-jre-headless.yaml

```diff
@@ -0,0 +1,160 @@
+package: openjdk-17-jre-headless
+
+essential:
+  - openjdk-17-jre-headless_copyright
+
+slices:
```

Comment:
We should probably leave a one-liner comment on top of each slice describing what it does.